### ISSUE TITLE
Option - Enable Smooth font not working anymore

### ISSUE THREAD

**Issue by Arczi008TV** [notepad-plus-plus/notepad-plus-plus#14954]
Something bad happened to Font smooting option. 
On old versions I could have aliased or antialiased fonts.
On new versions this option do nothing.  Unnoticable microscopic changes lol.

**Font smoothing in Notepad 8.1.4**
![ezgif com-animated-gif-maker](https://github.com/notepad-plus-plus/notepad-plus-plus/assets/49120706/ca0e0dac-7f27-4717-8720-ff6592210f4c)

**Font smoothing in Notepad 8.6.5**
![ezgif com-animated-gif-maker2](https://github.com/notepad-plus-plus/notepad-plus-plus/assets/49120706/226c96c7-ebbd-4225-9ced-230bd6161378)

**Comment by rdipardo** [comment]
I think the issue title means to say that __*dis*__&#x200D;abling font smoothing doesn't work anymore &mdash; specifically, not since v8.6, the first release to come with [Direct Write](https://learn.microsoft.com/windows/win32/directwrite/direct-write-portal) enabled [by default][0].

You can still disable it by going to `Settings` > `Preferences...` > `MISC.`:

<img width="480" alt="v8 6 5-no-D2D" src="https://github.com/notepad-plus-plus/notepad-plus-plus/assets/59004801/62a91e04-4045-4dc6-9eee-1625955b1b0b">

[0]: https://github.com/notepad-plus-plus/notepad-plus-plus/commit/975d29b30bfd624b6392541b133c2bd32f3212b0

**Comment by Arczi008TV** [comment]
Yes, you right, smoothing won't go away if enable smooth fonts is unchecked..
